Module/PSWordCloud.psd1

```diff
@@ -0,0 +1,129 @@
+#
+# Module manifest for module 'PSWordCloud'
+#
+# Generated by: Joel Sallow
+#
+# Generated on: 7/8/2018
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    RootModule           = 'PSWordCloud.psm1'
+
+    # Version number of this module.
+    ModuleVersion        = '2.0.0'
+
+    # Supported PSEditions
+    CompatiblePSEditions = @('Core')
```

Comment:
What prevents it from running on Windows PowerShell? Mostly just curious.

Reply:
Nothing huge, I just hate having to make my PSD1 UTF16LE and I'd rather depend on the newer libraries.

I could probably dial it back to PS Standard I guess. Those git diffs look awful though.

Reply:
Dealer's choice. It's your module so you get to choose. I just didn't see anything Core-specific that prevents you from doing it so I thought I'd ask.

Reply:
I fully expect a lot of requests to come in asking for it to be WinPS compatible.

Maybe I should just head them off before it becomes an annoyance... 🤔